server/container/util-java8/src/main/java/org/apache/james/util/OptionalUtils.java

```diff
@@ -52,7 +52,7 @@
             .isPresent();
     }
 
-    public static <T> boolean requestChanged(Optional<T> requestValue, Optional<T> storeValue) {
+    public static <T> boolean contentChangedCompareToStoreValue(Optional<T> requestValue, Optional<T> storeValue) {
```

Comment:
maybe you mean then contentIsDifferent(Optional<T> optional, Optional<T> reference) ?

Reply:
It has some strange when testing with data (empty, content) = true and (content, empty) = false
So the name is ok IMO